week-1-ts-ecosystem/w01e01-typescript-basics/src/logic.ts

```diff
@@ -0,0 +1,11 @@
+import {User} from './data'
```

Comment:
`logic.ts` zmieniłbym na `user.validators.ts` na przykład. to ich funkcjonalność